polaris-react/src/components/Modal/Modal.stories.tsx

```diff
@@ -58,17 +70,17 @@ export function WithPrimaryAction() {
   const discountLink = 'https://polaris.shopify.com/';
 
   const [active, setActive] = useState(true);
-  const node = useRef(null);
+  const node: RefObject<HTMLDivElement> = useRef(null);
 
   const handleClick = useCallback(() => {
-    node.current && node.current.input.focus();
+    node.current && node.current.querySelector('input')?.focus();
```

Comment:
We definitely don't want to recommend `querySelector`. The `useRef` approach is the correct one and we might want an additional `useRef` for the child.